books/acknowledgments.md.html

```diff
@@ -0,0 +1,76 @@
+<meta charset="utf-8">
+<!-- Markdeep: https://casual-effects.com/markdeep/ -->
+
+Acknowledgments
+====================================================================================================
+
+<div class="credit-list"> **Original Manuscript Help**
+
+  - Dave Hart
+  - Jean Buckley
+</div>
+
+<div class="credit-list"> **Web Release**
+
+  - Berna Kabadayı
+  - Lorenzo Mancini
+  - Lori Whippler Hollasch
+  - Ronald Wotzlaw
+</div>
+
+<div class="credit-list"> **Corrections and Improvements**
+
+  - Aaryaman Vasishta
+  - Andrew Kensler
+  - Apoorva Joshi
+  - Aras Pranckevičius
+  - Becker
+  - Ben Kerl
+  - Benjamin Summerton
+  - Bennett Hardwick
+  - Dan Drummond
+  - David Chambers
+  - David Hart
+  - Eric Haines
+  - Fabio Sancinetti
+  - Filipe Scur
+  - Frank He
+  - Gerrit Wessendorf
+  - Grue Debry
+  - Ingo Wald
+  - Jason Stone
+  - Jean Buckley
+  - Joey Cho
+  - Lorenzo Mancini
+  - Marcus Ottosson
+  - Matthew Heimlich
+  - Nakata Daisuke
+  - Paul Melis
+  - Phil Cristensen
+  - Ronald Wotzlaw
+  - Shaun P. Lee
+  - Tatsuya Ogawa
+  - Thiago Ize
+  - Vahan Sosoyan
+</div>
+
+<div class="credit-list"> **Tools**
+
+  <div class="indented">
+  Thanks to the team at [Limnu](https://limnu.com/) for help on the figures.
+
+  These books are entirely written in Morgan McGuire's fantastic and free
+  [Markdeep](https://casual-effects.com/markdeep/) library. To see what this looks like, view the
+  page source from your browser.
+
+  </div>
+</div>
+
+
+
+<!-- Markdeep: https://casual-effects.com/markdeep/ -->
+<link rel='stylesheet' href='../style/book.css'>
+<style class="fallback">body{visibility:hidden;white-space:pre;font-family:monospace}</style>
+<script src="markdeep.min.js"></script>
+<script src="https://casual-effects.com/markdeep/latest/markdeep.min.js"></script>
```

Comment:
Also got rid of wrong link issue +1

Reply:
Yeah, I closed that specific PR in favor of this one.